### ISSUE TITLE
about gt and Truelabel

### ISSUE THREAD

**Issue by yrj1409** [yuyang-long/SSA#4]
it seems that "gt=img_obj['TrueLabel'] - 1"

But why use **gt** in "attack.py" to generate adversarial examples but use **gt+1** in "verify.py" to get accuracy? 

## **attack.py**
loss = F.cross_entropy(output_v3, gt)

## **verify.py**
sum += (model(images)[0].argmax(1) != (gt+1)).detach().sum().cpu()

**Comment by yuyang-long** [comment]
The output dimension of pretrainedmodels' model is 1000 x 1, while tf_xxx model is 1001 x 1.
